base/applications/rapps/gui.cpp

```diff
@@ -750,7 +750,7 @@ HWND
 CMainWindow::Create()
 {
     CStringW szWindowName;
-    szWindowName.LoadStringW(IDS_APPTITLE);
+    szWindowName.LoadStringW(m_bAppwizMode ? IDS_APPWIZ_TITLE : IDS_APPTITLE);
```

Comment:
could be changed by:
```c++
const CStringW szWindowName(MAKEINTRESOURCEW(m_bAppwizMode ? IDS_APPWIZ_TITLE : IDS_APPTITLE));
```

Reply:
Does this have an actual effect on the generated code?

Reply:
Probably not much; with what I propose it would be with the constructor that the LoadStringW would be called.